src/components/react/Range/index.js

```diff
@@ -170,8 +170,8 @@ class Range extends Component {
     return (
       <div
         className={classes('ugli-range-container', { rtl, vertical })}
-        onTouchStart={e => this.touchStartHandler(e)}
-        onTouchEnd={() => this.touchEndHandler()}
+        onTouchStart={this.touchStartHandler}
+        onTouchEnd={this.touchEndHandler}
         onTouchMove={throttle(this.touchMoveHandler)}
```

Comment:
`throttle(this.touchMoveHandler)` this will generate a new throttle function every time react render, I think move this outside the `render` function is better.
Like:
```js
class Example {
  touchMoveHandler = throttle(e => {
     this.props.onSlide(this.value)
     this.touchHandler(e)
   })
}
```

Reply:
And you added a /* eslint-disable */ at the top of the file, I think it is better to remove it.